Makefile

```diff
@@ -2,6 +2,19 @@
 build :
 	dune build
 
+.PHONY : npm
+npm : npm-build
+	esy release
+
+.PHONY : npm-build
+npm-build :
+	esy install
+	esy build
+
+.PHONY : npm-test
+npm-test :
```

Comment:
This seems very meta (ie a make command that calls esy that alls make again). How is it different to `make test`?

Reply:
@rizo This runs `make test` in the esy environment. It doesn't seem too bad.

Reply:
Alternatively, we can just call `esy make test`, but I fear that from looking at the files we currently have it won't be obvious that this should be run (at some point) to make sure the build works as an `npm` package.

We can also document it in the contribution guidelines.